services/core-api/app/api/mines/documents/resources/mine_document_version_resource.py

```diff
@@ -49,8 +57,16 @@ def post(self, mine_guid, mine_document_guid):
         if mine_document.is_archived:
             raise BadRequest('Cannot create new version of archived document')
 
-        return DocumentManagerService.initializeFileVersionUploadWithDocumentManager(
+        resp = DocumentManagerService.initializeFileVersionUploadWithDocumentManager(
             request, mine_document)
+        project_name = Project.find_by_project_guid(project_guid)
+
+        if resp:
+            renotify_hours = 24
+            trigger_notification(f'File(s) in project {project_name} has been updated for mine {mine.mine_name}.',
```

Comment:
I think this should be in the `MineDocumentVersionListResource` not in the `MineDocumentVersionUploadResource`. `MineDocumentVersionUploadResource` used when you first initialize an upload, where `MineDocumentVersionListResource` is used when the upload has completed - and we probably want to only send the notification then

Reply:
Updated accordingly